packages/gx-agent/src/agent/GXComplianceClient.ts

```diff
@@ -324,12 +332,14 @@ export class GXComplianceClient implements IAgentPlugin {
         keyRef: args.keyRef,
         verifiableCredentials: [args.selfDescriptionVC, args.complianceVC],
         domain: convertDidWebToHost(participantDid),
+        persist: args.persist ? args.persist : false,
       },
       context
     )
     const verifiableCredentialResponse = (await this.acquireComplianceCredential(
       {
         verifiablePresentation: uniqueVP.verifiablePresentation,
+        show: args.show ? args.show : false,
```

Comment:
Not sure why this is needed everywhere, simply pass in args.show

Reply:
changed to just: `args.persist`